.github/ISSUE_TEMPLATE/05-dev-days-project.yml

```diff
@@ -0,0 +1,73 @@
+name: Indicate your project is participating in Dev Days
+description: | 
+  For hosted projects only, use this form to indicate that your project will be participating in Dev Days.
+title: "Dev Days 2024 Project Application - PROJECTNAME"
+labels: ["help wanted"]
+assignees: 
+  - yarille
+  - jmertic
+  - carolalynn
+  - lgritz
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ![](https://artwork.aswf.io/other/dev-days/2024/color/dev-days-2024-color.svg)
+
+        Thank you for your interest in being a project wanting to participate in Dev Days 2024!
+        For hosted projects only, use this form to indicate that your project will be participating in Dev Days.
+
+        Please update `PROJECTNAME` in the title with your project name.
+  - type: input
+    attributes:
+      label: URL to your Dev Days 2024 landing page
+      description: |
+        This URL can be a webpage, wiki page, or file in your code repository that outlines...
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: URL of issues for participants to work on
+      description: |
+        Projects should label issues with 'good first issue' and 'help wanted' and ensure 
+        they are unassigned and no more than one year old. To validate, go to 
+        [CLOTributor](https://clotributor.dev/search?foundation=aswf&good_first_issue=true).
+
+        These issues should align with the Academy Software Foundation TAC's
+        [best practices for good first issues]()
```

Comment:
@carolalynn @lgritz This resource we still need to create - we could do a wiki page to specific for Dev Days to start and then roll into the other TAC policies over time.